src/Microsoft.Azure.WebJobs.ServiceBus/Triggers/ServiceBusTriggerBinding.cs

```diff
@@ -122,6 +127,75 @@ public Task<IListener> CreateListenerAsync(ListenerFactoryContext context)
             return factory.CreateAsync(context.CancellationToken);
         }
 
+        internal static IReadOnlyDictionary<string, Type> CreateBindingDataContract(IReadOnlyDictionary<string, Type> argumentBindingContract)
+        {
+            var contract = new Dictionary<string, Type>(StringComparer.OrdinalIgnoreCase);
+            contract.Add("DeliveryCount", typeof(int));
```

Comment:
I picked a subset of all the properties to add, based on what I think might be useful. I didn't add everything. I did add the ones customers have explicitly asked for though. Let me know if you think any others should be added.